packages/blade/src/components/Radio/RadioGroup/RadioGroup.tsx

```diff
@@ -94,12 +103,16 @@ const RadioGroup = ({
     onChange,
     validationState,
     value,
+    size,
   });
 
+  const { theme } = useTheme();
+  const { matchedDeviceType } = useBreakpoint({ breakpoints: theme.breakpoints });
   const showError = validationState === 'error' && errorText;
   const showHelpText = !showError && helpText;
   const accessibilityText = `,${showError ? errorText : ''} ${showHelpText ? helpText : ''}`;
   const isReactNative = getPlatformType() === 'react-native';
+  const gap = radioSizes.group.gap[size][matchedDeviceType];
```

Comment:
Lets remove `gap` please while we're working on this PR otherwise we won't revisit this and it will remain broken on react native

Reply:
What would be the equivalent of the gap here? 

I mean how should we approach it so that in react-native the gaps can be in between. Does the owl selector work in RN 🤔

Reply:
No, we'll have to stick to using margins :(